EIPS/eip-4626.md

```diff
@@ -214,7 +214,7 @@ Mints `shares` Vault shares to `receiver` by depositing exactly `amount` of unde
 
 MUST emit the `Deposit` event.
 
-MAY support an additional flow in which the underlying tokens are owned by the Vault contract before the `deposit` execution, and are accounted for during `deposit`.
+MUST support ERC-20 `approve` / `transferFrom` on `asset` as a deposit flow. MAY support an additional flow in which the underlying tokens are owned by the Vault contract before the `deposit` execution, and are accounted for during `deposit`.
```

Comment:
Hot tip: Markdown renders separate lines with no extra space between them as the same paragraph.
```suggestion
MUST support ERC-20 `approve` / `transferFrom` on `asset` as a deposit flow.
MAY support an additional flow in which the underlying tokens are owned by the Vault contract before the `deposit` execution, and are accounted for during `deposit`.
```